super-intel.c

```diff
@@ -2783,30 +2795,23 @@ static int detail_platform_imsm(int verbose, int enumerate_only, char *controlle
 		return result;
 	}
 
-	const struct orom_entry *entry;
-
 	for (entry = orom_entries; entry; entry = entry->next) {
+		print_imsm_capability(entry);
+
 		if (entry->type == SYS_DEV_VMD) {
-			print_imsm_capability(&entry->orom);
-			printf(" 3rd party NVMe :%s supported\n",
-			    imsm_orom_has_tpv_support(&entry->orom)?"":" not");
 			for (hba = list; hba; hba = hba->next) {
 				if (hba->type == SYS_DEV_VMD) {
 					char buf[PATH_MAX];
+
 					printf(" I/O Controller : %s (%s)\n",
 						vmd_domain_to_controller(hba, buf), get_sys_dev_type(hba->type));
```

Comment:
To long line. You can reduce idenation here
```
		if (hba->type != SYS_DEV_VMD)
					continue;
		char buf[PATH_MAX];
		printf(" I/O Controller : %s (%s)\n",
				     vmd_domain_to_controller(hba, buf),
				     get_sys_dev_type(hba->type));
		print_nvme_info(hba);
```

Reply:
Done, and I squashed vmd and nvme printing,